decompressed/base/etc/init.d/rootdevice

```diff
@@ -359,6 +372,7 @@ boot() {
     fi
   fi
 
-  check_bank
+  #TODO: uncomment this when ready
+  #is_dual_bank && check_and_apply_obp
```

Comment:
Questa va fatto dopo check_root, dopo la pausa e prima di procedere all'installazione

Reply:
perchè prima dell'installazione? l'idea era di fargli spostare poi la GUI già installata sull'overlay di bank2

Reply:
no non funzionerebbe, devi prima metterti su OBP e poi applicare la mod altrimenti stai eseguendo l'installazione sul bank plan pericoloso, non avrebbe senso

Reply:
Ma comunque tu qui hai già scompattato il tar della mod al suo posto, il pacchetto in tmp da questo punto in avanti non ti occorre più. Al riavvio questo script non trova la flag che segna la gui come installata e la installa.

Quella cosa che avete qui in questo script fa il setup della mod già scompattata, più che un install è un post-install che la attiva, prima di eseguirlo la mod oltre ad essere stata copiata nell'overlay non si è ancora applicata effettivamente.

Reply:
il tar ci serve per intero per poterlo (ri)applicare negli upgrade firmware, per questo l'install ce lo porta in /root (e se online ce lo riscarica e lo ficca lì)

l'alternativa che ci vedo è estrarre solo rootdevice (o l'intero pacchetto base), mettere il pacchetto il /root e riavviare in modo che sia l'estrazione che l'install verra' fatta dopo aver attivato al 100% l'OBP

Reply:
Allora sì, estrai solo il pacchetto base e rimanda l'estrazione del resto al dopo reboot. Ti tornerà utile quando mai vorrai abilitare l'estrazione di quei file nel secondo overlay.

Reply:
Ripensandoci, troppo complicato, ad ora la GUI viene già estratta prima di lanciare alcuno script inteligente, quindi cosi come è ora, spostando l'obp prima dell'install, l'installazione vera e propria della GUI partirà al reboot.

il pacchetto permanente in /root andrà perso nel caso di installazioni offline ma è un compromesso accettabile e che non dovrebbe causare conseguenze particolarmente gravi